### DISCUSSION TITLE
[Feature Request]: PicoW WiFi AP for management.

### DISCUSSION THREAD

**Discussion by madeofstown** [meshtastic/firmware#3824] [Ideas]
### Platform

NRF52

### Description

This feature is for the RaspberryPi Pico W not NRF52.

The PicoW is a nice lower power device that I would like to deploy in a remote location that would require a ladder to access. As of now, the only way to manage it would be to use the admin channel, because of the lack of support for the Bluetooth radio. I have found the admin channel somewhat unreliable and unresponsive.

I'm proposing that the PicoW should create a WiFi access point when it has it's WiFi radio enabled and there is no access point with the settings configured in range for a specified time. 

If that just isn't feasible, perhaps the PicoW should not entirely give up on trying to connect to WiFi. Possibly have it check for the configured network every 15 minutes or so indefinitely. That way, I could at least broadcast an AP from my phone that the PicoW would eventually connect to if I stood around long enough.

**Comment by caveman99** [comment]
We removed AP mode a while back, because it was raising wrong expectations. You can set up the pico through the serial connection with the CLI or the hosted web interface.
